app/views/govuk_publishing_components/audit/_components.html.erb

```diff
@@ -66,7 +66,26 @@
                 <%= component[:component] %> (<%= component[:count] %>)
               </dt>
               <dd class="govuk-summary-list__value">
-                <%= component[:list] %>
+                <p><%= component[:list] %></p>
```

Comment:
Could the list of applications that the component is used be shown only once in the `details` element?

For example: 

![image](https://user-images.githubusercontent.com/1732331/178231849-0eafb3e7-65d0-4c87-b4be-5246f422884f.png)

Reply:
Oh yes, that's much more sensible, thanks. Have changed.